src/constants/indicators.js

```diff
@@ -299,4 +329,424 @@ export default {
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Land area that is parkland, % of gross land area',
+  },
+  greenhouseGasEmissions: {
+    name: 'Greenhouse gas emissions',
+    shortDescription: 'Kilograms of greenhouse gas emissions per capita per year',
+    longDescription: 'Kilograms of greenhouse gas emissions per capita per year',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0a',
+    contextual: false,
+    source: 'Greenhouse gas emissions  per capita, kilograms per capita per year',
+  },
+  accessToGreenSpace: {
+    name: 'Access to green space',
+    shortDescription: 'Percentage of dwellings with access to green space',
+    longDescription: 'Percentage of dwellings with access to green space',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Access to green space, % of dwellings',
+  },
+  airPollutionPm10: {
+    name: 'Air pollution, >PM10',
+    shortDescription: 'Particle pollution greater than PM10',
+    longDescription: 'Air quality, annual mean: particle pollution greater than PM10',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0a',
+    contextual: false,
+    source: 'Air quality, Annual mean, >PM10, ug/m3',
+  },
+  airPollutionPm2point5: {
+    name: 'Air pollution, <PM2.5',
+    shortDescription: 'Particle pollution less than PM2.5',
+    longDescription: 'Air quality, annual mean: particle pollution less than PM2.5',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0a',
+    contextual: false,
+    source: 'Air quality, Annual mean, <PM2.5, ug/m3',
+  },
+  energyEfficiencyOfBuildings: {
+    name: 'Energy effiency of buildings',
+    shortDescription: 'Average energy efficiency rating of commercial buildings',
+    longDescription: 'Average energy efficiency rating of commercial buildings',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0',
+    contextual: false,
+    source: 'Building energy efficiency, Average rating',
+  },
+  volunteeringRate: {
+    name: 'Volunteering rate',
+    shortDescription: 'Percentage of population over 15 years who volunteer',
+    longDescription: 'Percentage of population over 15 years who volunteer',
+    dataType: DATA_TYPES.NUMBER,
+    aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+    format: '0[.]0%',
+    contextual: false,
+    source: 'Number of volunteers, All persons, % of Persons 15 or over',
+  },
+  // perceivedSafety: {
+  //   name: 'Perceived safety',
+  //   shortDescription: 'TODO',
+  //   longDescription: 'TODO',
+  //   dataType: DATA_TYPES.NUMBER,
+  //   aggregationMethod: AGGREGATION_METHODS.MEDIAN,
+  //   format: '0[.]0%',
+  //   contextual: false,
+  //   source: 'TODO',
+  // },
```

Comment:
Commented out code can go?

Reply:
This is temporarily commented out until James can get the data in. The data needs to be there but isn't currently available in the csv.